app/controllers/concerns/fraud_review_concern.rb

```diff
@@ -1,6 +1,8 @@
 module FraudReviewConcern
   extend ActiveSupport::Concern
 
+  delegate :fraud_check_failed?, :fraud_review_pending?, :fraud_rejection?, to: :fraud_review_check
```

Comment:
Great way to encapsulate this! So `:fraud_review_check` can refer to an instance variable? TIL about delegate.